source/common/secret/secret_manager_impl.h

```diff
@@ -14,13 +14,11 @@ class SecretManagerImpl : public SecretManager, Logger::Loggable<Logger::Id::ups
 public:
   SecretManagerImpl(){};
 
-  void addOrUpdateSecret(SecretSharedPtr secret) override;
-  const SecretSharedPtr findSecret(const std::string& name) const override;
-
 private:
-  typedef std::unordered_map<std::string, SecretSharedPtr> SecretSharedPtrMap;
+  void addOrUpdateSecret(const std::string& type, const SecretSharedPtr& secret) override;
+  const SecretSharedPtr findSecret(const std::string& type, const std::string& name) const override;
 
-  SecretSharedPtrMap static_secrets_;
+  std::unordered_map<std::string, std::unordered_map<std::string, SecretSharedPtr>> secrets_;
```

Comment:
please add comment about two dimension maps, what is the first level key, what is the second level key and its value

Reply:
Added comment